pages/games.vue

```diff
@@ -70,8 +70,13 @@ export default {
             async handler(newSeason) {
                 this.season = Number(newSeason) || 3;
 
-                this.games = await Http.for('games/season').get(this.season);
+                const games = await Http.for('games/season').get(this.season);
```

Comment:
@samdharris Do you think you could change this request to use the axios object instead of the HTTP? We are moving away from the HTTP objective as and when we work on a area, just switch it over?

You should be able to find other examples in the project, something like `this.$axios.get(utl)`?